### PR TITLE
Bump Binaryen to 120

### PR DESCRIPTION
**Pull request by aheejin**
https://github.com/WebAssembly/binaryen/pull/7153 updated Binaryen version to 121, but bumping the version to 121 here does not pass the CI (because Binaryen version is behind by 2), so this updates the version to 120 for the moment.

### CODE REVIEW

**Review by sbc100 — APPROVED** [review]

**Comment by sbc100** [comment]
Feel free to land this without waiting for CI if it helps unblock the rollers.

**Review by kripken — APPROVED** [review]

**Pull request merged**
